components/next_steps_view/index.ts

```diff
@@ -22,12 +26,16 @@ function makeMapStateToProps() {
     const getCategory = makeGetCategory();
 
     return (state: GlobalState) => {
+        const teamId = LocalStorageStore.getPreviousTeamId(getCurrentUserId(state));
+        const team = getTeam(state, teamId || '');
         return {
             currentUser: getCurrentUser(state),
             isAdmin: isCurrentUserSystemAdmin(state),
             preferences: getCategory(state, Preferences.RECOMMENDED_NEXT_STEPS),
             steps: getSteps(state),
             isFirstAdmin: isFirstAdmin(state),
+            isCloud: getLicense(state).Cloud === 'true',
```

Comment:
All onboarding section is only shown for cloud so not sure this check is needed here